lib/compress.js

```diff
@@ -706,9 +706,9 @@ merge(Compressor.prototype, {
         return x;
     };
 
-    function is_iife_call(node) {
+    function is_iife_call(node, traditional) {
         if (node instanceof AST_Call && !(node instanceof AST_New)) {
-            return is_func_expr(node.expression) || is_iife_call(node.expression);
+            return is_func_expr(node.expression, traditional) || is_iife_call(node.expression);
```

Comment:
Looks like `is_iife_call()` will always be passing `true` here. If so, may be do away with `traditional` for `is_iife_call()` and just hardcode `true` here?